apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/wpcom-nux-tour.js

```diff
@@ -0,0 +1,92 @@
+/*** THIS MUST BE THE FIRST THING EVALUATED IN THIS SCRIPT *****/
+import './public-path';
+
+/**
+ * Internal dependencies
+ */
+import WelcomeTourCard from './tour-card';
+import getTourContent from './tour-content';
+import './style-tour.scss';
+
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { Icon, chevronUp } from '@wordpress/icons';
+// TODO: fix issue with expand icon not found.  Probably to do with old icon lib version
+
+import { createPortal, useEffect, useState } from '@wordpress/element';
+import { registerPlugin } from '@wordpress/plugins';
+
+function LaunchWpcomNuxTour() {
+	// Create parent for welcome tour portal
+	const portalParent = document.createElement( 'div' );
+	portalParent.classList.add( 'wpcom-editor-welcome-tour-portal-parent' );
+	useEffect( () => {
+		document.body.appendChild( portalParent );
+		return () => {
```

Comment:
This portalParent is not being removed from the dom.  I think maybe there are some things I'm not understanding about Portals/React.  I don't see how this `LaunchWpcomNuxTour` component ever unmounts. I tried to pick through the [Editor NavSidebar](https://github.com/Automattic/wp-calypso/blob/d2952aced281538500583873714bdb25f7fb6d0f/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/attach-sidebar.tsx#L41) for guidance, but didn't really help since the sidebar is always in in the dom (isn't it)?  Would love to discuss with someone wiser than I. 🧠

Reply:
I'm no expert, but the contents of your `useEffect` method look good.

 I don't see where you're unmounting, or, in other words, hiding/showing the LaunchWpcomNuxTour  component. 

There might need to be some conditional state change required? E.g., `someStateVar && <LaunchWpcomNuxTour />` ??

Reply:
Actually, after some additional thought/reading, I don't think the `portalParent` needs to be removed from the DOM.  Many react examples create a static div for attaching portal components.   Even in the calypso Block Editor dom I found `<div class="ReactModalPortal"></div> ` (but I failed to locate the code that inserts this div 😞 ): 
![Screen Shot 2020-11-20 at 15 48 18](https://user-images.githubusercontent.com/5665959/99760537-06673980-2b48-11eb-8045-60a32ce618e8.png)



As far as unmounting <LaunchWpcomNuxTour/> I think I was thinking about it wrong.  It is render via 
```
registerPlugin( 'wpcom-block-editor-nux', {
	render: () => <LaunchWpcomNuxTour />,
} );
```

Reply:
As for my concerns of unmounting `<LaunchWpcomNuxTour/>` I think I was thinking about it wrong, given that it is rendered within the `registerPlugin`/wp system.

This React component is rendered via 
```
registerPlugin( 'wpcom-block-editor-nux', {
	render: () => <LaunchWpcomNuxTour />,
} );
```
and I think the component is then meant to stay mounted.  Because it is the root of the plugin, where the React component `<LaunchWpcomNuxTour />` _is the plugin_.   Am I describing all of this correctly?